pkgs/development/python-modules/oelint-parser/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pythonRelaxDepsHook
+, regex
+}:
+
+buildPythonPackage rec {
+  pname = "oelint-parser";
+  version = "2.10.5";
+
+  src = fetchPypi {
+    pname = "oelint_parser";
+    inherit version;
+    hash = "sha256-k80qW4uTObSpCjJFqS9xNX58jkUwep8xnHFdNVRi5TY=";
+  };
+
+  disabled = pythonOlder "3.7";
+
+  nativeBuildInputs = [ pythonRelaxDepsHook ];
+  pythonRelaxDeps = true;
+
+  propagatedBuildInputs = [ regex ];
+
+  pythonImportsCheck = [ "oelint_parser" ];
+
+  # Currently fails due to various issues
+  doCheck = false;
```

Comment:
I tried that, but did not realize I that `pytestCheckHook` must come _after_ all other `pytest`-dependencies in `checkInputs`, which otherwise result in some transient dependencies. Not exactly obvious 😓.